tests/integration/test_fim/test_registry/test_registry_report_changes/test_disk_quota/test_registry_disk_quota_bigger_file_limit.py

```diff
@@ -0,0 +1,207 @@
+                               calculate_registry_diff_paths, registry_value_create, registry_value_update,
+                               registry_value_delete, registry_parser, create_values_content)
+from wazuh_testing.fim_module.fim_variables import (WINDOWS_HKEY_LOCAL_MACHINE, MONITORED_KEY, MONITORED_KEY_2,
+                                                    ERR_MSG_DISK_QUOTA_MUST_BE_GREATER, ERR_MSG_WRONG_DISK_QUOTA_VALUE,
+                                                    CB_FILE_SIZE_LIMIT_BIGGER_THAN_DISK_QUOTA, 
+                                                    SIZE_LIMIT_CONFIGURED_VALUE, ERR_MSG_CONTENT_CHANGES_EMPTY,
+                                                    ERR_MSG_CONTENT_CHANGES_NOT_EMPTY)
+from wazuh_testing.tools.configuration import load_wazuh_configurations
+from wazuh_testing.tools.monitoring import FileMonitor, callback_generator
+
+# Marks
+
+pytestmark = [pytest.mark.win32, pytest.mark.tier(level=1)]
+
+# Variables
+
+key = WINDOWS_HKEY_LOCAL_MACHINE
+sub_key_1 = MONITORED_KEY
+sub_key_2 = MONITORED_KEY_2
+
+test_regs = [os.path.join(key, sub_key_1), os.path.join(key, sub_key_2)]
+reg1, reg2 = test_regs
+test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), "data")
+wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
+size_limit_configured = SIZE_LIMIT_CONFIGURED_VALUE
+
+# Configurations
+
+p, m = generate_params(modes=["scheduled"], extra_params={
+                                                        "WINDOWS_REGISTRY_1": reg1,
+                                                        "WINDOWS_REGISTRY_2": reg2,
+                                                        "FILE_SIZE_ENABLED": "yes",
+                                                        "FILE_SIZE_LIMIT": "10KB",
+                                                        "DISK_QUOTA_ENABLED": "yes",
+                                                        "DISK_QUOTA_LIMIT": "1KB",
+                                                        })
+
+configurations_path = os.path.join(test_data_path, "wazuh_registry_report_changes_limits_quota.yaml")
+configurations = load_wazuh_configurations(configurations_path, __name__, params=p, metadata=m)
+
+
+# Fixtures
+
+
+@pytest.fixture(scope="module", params=configurations)
+def get_configuration(request):
+    """Get configurations from the module."""
+    return request.param
+
+
+@pytest.mark.parametrize("size", [(8 * 1024), (32 * 1024)])
+@pytest.mark.parametrize(
+    "key, subkey, arch, value_name",
+    [
+        (key, sub_key_1, KEY_WOW64_64KEY, "some_value"),
+        (key, sub_key_1, KEY_WOW64_32KEY, "some_value"),
+        (key, sub_key_2, KEY_WOW64_64KEY, "some_value"),
+    ],
+)
+def test_disk_quota_values(key, subkey, arch, value_name, size, get_configuration, configure_environment,
+                           restart_syscheckd, wait_for_fim_start):
+    """
+    description: Check if the 'wazuh-syscheckd' daemon sets the 'disk_quotap' limit to be equal to the
```

Comment:
```suggestion
    description: Check if the 'wazuh-syscheckd' daemon sets the 'disk_quota' limit to be equal to the
```

Reply:
Fixed in commit https://github.com/wazuh/wazuh-qa/pull/2478/commits/50987ed612082b485cdda2af0c6f3c1aeceffed0